src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
+@import "./variables";
+@import "./stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  color: #060b35;
+
+  h1,
+  h2,
+  p {
+    margin: 0;
+  }
+
+  .product-card {
```

Comment:
`.product-card` is an independent BEM class and this class should not be nested in other styles